packages/v4/src/content/get-started/about.md

```diff
@@ -26,7 +26,7 @@ Style guidelines define foundational elements of the design system, like color,
 ### Usage and behavior
 Usage and behavior guidelines communicate standards and best practices for common design patterns like navigation, dashboards, or forms.
 
-### Content
+### UX writing
 Content guidelines provide principles and best practices around writing for user experience along with general voice and style guidance.
 
 [**View design guidelines**](/guidelines/colors) <i className="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
```

Comment:
This link has been updated to this one: https://www.patternfly.org/v4/ux-writing/about